web/packages/extension/src/utils.ts

```diff
@@ -95,6 +104,7 @@ function promisifyStorageArea(
 
 if (typeof chrome !== "undefined") {
```

Comment:
Oh. I think we put these `if` in the wrong order. Firefox has both `browser` and `chrome`, so we would always use `chrome` even in Firefox? Not something we should fix here.

Reply:
Yeah, I'm guessing this order might have been from before all the other browsers added support for the `chrome` namespace, although I don't know when that is as I've not found an MDN page with support info for the `chrome` and `browser` namespaces themselves.